instructors/4-gis-slides.qmd

```diff
@@ -468,39 +436,44 @@ st_crs(old_buildings)
 buffer_old_buildings <- 
   st_buffer(x = old_buildings, dist = distance)
  
- ggplot(data = buffer_old_buildings) + geom_sf()
- 
+ggplot(data = buffer_old_buildings) + 
+  geom_sf() +   
+  coord_sf(datum = st_crs(28992))
+
 ```
 
 ## Union
 
 Now, we have overlapping buffers.
 
-To fuse (or dissolve) them into a **unique** conservation zone/polygon,
-we use `sf::st_union` with `st_cast()` to explicit the type of object
-(*POLYGON* instead of the default *MULTIPOLYGON*), and `st_as_sf()` to
-transform the polygon into an `sf` object.
+We would rather create a **unique** conservation zone rather than overlapping ones in that case. So we have to fuse the overlapping buffers into one polygon. This operation is called union and the corresponding function is st_union().
 
 ```{r union}
 single_old_buffer <- st_union(buffer_old_buildings) %>%
    st_cast(to = "POLYGON") %>%
   st_as_sf() 
```

Comment:
A small detail... I see a few inconsistencies in the indentations throughout the presentation. Not a big issue on the slides, but it would be less confusing if the same two-space indentation is used for every level.